tests/system/test_limit_eps/data/block_changed.xml

```diff
@@ -0,0 +1,10 @@
+<ossec_config>
```

Comment:
This configuration and the ones below should be defined in a .yaml file like in [this](https://github.com/wazuh/wazuh/blob/master/tests/integration/test_api/test_config/test_upload_configuration/data/test_cases/cases_upload_configuration.yaml) case.